### PR TITLE
Mrsj/formatpages

### PR DESCRIPTION
**Pull request by matthewrsj**
Significant changes made to dougfir-pelican-theme, including:

* Added images to path
* Modified and added to CSS to improve formatting
* Added blog-sidebar.html to includes for custom sidebar when in article or blog page
* Began to work on cleaning up CSS by adding 
  - main.css
  - header.css 
  - footer.css
  - mainmenu.css 
  - sidebar.css

  I began populating these with CSS from the old styles but they are not yet included in the HTML.

### CODE REVIEW

**Comment by matthewrsj** [comment]
@Kennric

File: static/css/styles.css [hunk 1]
```diff
@@ -4813,7 +4837,7 @@ a:hover, a:active, a:focus {
 
 ul li {
   line-height: 20px;
-  list-style-image: none;
+  list-style-image: url(http://osuosl.org/sites/wwwstaging.osuosl.org/themes/doug_fir/images/bullet.png);
```
Comment (ramereth) [inline static/css/styles.css]: This should NOT be using a direct URL like this and instead should be a relative URL that points to a file hosted in this theme.

**Comment by ramereth** [comment]
Why does the theme include site specific images? That should only be in the ousosl-pelican repo, not the theme itself. The theme should only contain the common images that are needed for the theme itself.

File: static/css/header.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+#osu-top-hat #osu-search-top-hat-form label, #osu-top-hat #osu-search-top-hat-form input {
+    font-size: 90%;
+}
+#osu-top-hat #osu-search-top-hat-form label {
+    color: #fff;
+    margin-top: 1em;
+    padding: 0 0.2em;
+}
+#osu-top-hat #osu-search-top-hat-form .form-submit {
+    background: url("/theme/img/search_icon_white.png") no-repeat scroll 10px 4px #c34500;
```
Comment (ramereth) [inline static/css/header.css]: This is missing when you load the page (404's).

**Comment by Kennric** [comment]
search removal looks fine, no errors intorduced.The code here looks good, but I can't comment on the content as much. +1

**Comment by matthewrsj** [comment]
@jordane

**Comment by ramereth** [comment]
Good enough for now. I'm sure there are more edits to be fixed but this PR is big enough as it is. Great work! :+1:

**Pull request merged**